corehq/tests/noseplugins/cmdline_params.py

```diff
@@ -3,6 +3,17 @@
 """
 from nose.plugins import Plugin
 
+REUSE_DB_HELP = """
+To be used in conjunction with the environment variable REUSE_DB=1.
+reset: Drop existing test dbs, then create and migrate new ones, but do not
+    teardown after running tests. This is convenient when the existing databases
+    are outdated and need to be rebuilt.
+flush: Flush all objects from the old test databases before running tests.
+    Much faster than `reset`.
+migrate: Migrate the test databases before running tests.
+    teardown: Skip database setup; do normal teardown after running tests.
```

Comment:
Yeah, it concats all whitespace to a single space.  I did this just to make it easier to read in the file.  I saw some stack overflow posts on writing your own renderer which preserves whitespace, but I'm not about to do that.